api_server.py

```
@@ -74,6 +87,29 @@ def format_hours(minutes: Optional[int]) -> Optional[str]:
     return f"{minutes / 60.0:.2f}h"
 
 
+def format_availability_data(row_end_time: Any, now: datetime) -> Dict:
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `row_end_time`

(ANN401)